src/txorecord.rs

```diff
@@ -29,7 +28,7 @@ pub enum TxORecordError {
     #[error("Failed to retrieve UTXO with ID {0:?}.")]
     FailedToRetrieveOutputWithId(String),
     #[error("Failed to spend inputs: {0:?}.")]
-    FailedToSpendInputs(Box<NauError>),
+    FailedToSpendInputs(#[from] Box<NauError>),
```

Comment:
You can prolly remove this after following my above comment.

Reply:
Oh. Meant the entire variant. I don't think it's being used anywhere anymore. Plus it's wrapping an higher type, which shouldn't happen mostly (I think there might be a place in the code that still does, which is a smell, but I'm not too worried about it).